### ISSUE TITLE
Timestamp is missing in time series data generator

### ISSUE THREAD

**Issue by Manonandan** [HPI-Information-Systems/gutentag#37]
This doesn't return a dataset with a timestamp. I can add my own timestamps. But it would be better I could configure the start date and time and get the timestamp for each datapoint.

**Comment by CodeLionX** [comment]
Yes, GutenTAG only produces the time series' values because it does not support variable sampling frequencies. We assume equidistant time steps. You can add the desired index or timestamps to the time series afterward.

What does your use case look like?
